### ISSUE TITLE
error on package updates needed

### ISSUE THREAD

**Issue by gabeio** [tjunnone/npm-check-updates#10]
it would be super nice to be able to use this as a build check to make sure everything you are working with is the most up to date version and to add a way to ignore certain packages in cases that the newer ones are broken/not available via npm... etc
```
--ignore [lodash,express]

if(updates needed)||(updates are ignored)
    process.exit(1)
```

**Comment by metaraine** [comment]
Available in v2.0.0-alpha5

### Install

`npm install -g npm-check-updates@unstable`

### Usage

--error-level 1: Return with exit code 0 if there are no errors. 
--error-level 2: Return with exit code 0 if there are no errors **and** no modules need to be updated.

```sh
$ npm-check-updates -e 2
```

**Comment by gabeio** [comment]
sounds good to me :smile: :+1:
